venom/src/test/java/com/github/venom/service/ServiceDelegateTest.kt

```diff
@@ -26,17 +28,20 @@ class ServiceDelegateTest {
     @Test
     fun startService_startServiceIfNotRunning() {
         setRunningState(false)
-        delegate.startService()
```

Comment:
Could we have a blank line after `setRunningState`? We follow this convention to visually separate the setup part of the test from the action and assertion.